build.gradle.kts

```diff
@@ -0,0 +1,55 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.6.20"
+    application
+
+    id("org.jlleitschuh.gradle.ktlint") version "10.2.1"
+    id("maven-publish")
+}
+
+group = "com.flagsmith"
+version = "1.0.0"
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencies {
+    testImplementation(kotlin("test"))
+
+    // https://mvnrepository.com/artifact/com.flagsmith/flagsmith-java-client
+    api("com.flagsmith:flagsmith-java-client:5.0.0") {
```

Comment:
I guess this client will fail to build until the java client exists in the maven repository?

Reply:
Correct. This probably why Github actions won't run until the release has been made for Java client 5.0.0.